Source/Core/DolphinQt2/Translate.h

```diff
@@ -0,0 +1,270 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QString>
+#include <QTranslator>
+
+class DQTranslator : public QTranslator
+{
+	public:
```

Comment:
`public` should be at the same level of indentation as the braces also the class should be made final